src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -35,10 +35,17 @@ public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_TAG);
 
+        if (!arePrefixesPresent(argMultimap, PREFIX_EMAIL)) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    AddCommand.MESSAGE_USAGE)); //Need to change the MESSAGE_USAGE.
```

Comment:
remove such un-informative comments. if you want you can use the github comments (like this one) or use commit messages in the VCS to give additional info not related to the application